airbyte-integrations/connectors/destination-redshift/src/test-integration/kotlin/io/airbyte/integrations/destination/redshift/RedshiftS3StagingStorageOperationTest.kt

```diff
@@ -197,6 +234,22 @@ class RedshiftS3StagingStorageOperationTest {
         }
     }
 
+    private fun getRedshiftObjects(dropCascade: Boolean): Pair<RedshiftStagingStorageOperation, RedshiftDestinationHandler> {
```

Comment:
this method name sucks, please suggest something better 😓

Reply:
the split of different objects sucks. It is what it is

Reply:
actually, a simple one would be to expose the stagingOperation.handler, so you can return only the storageOperation

Reply:
derp. I forgot that we already have the underlying jdbcDatabase object >.> so we only need to return the storage op object anyway